backend/src/schema/resolvers/moderation.js

```diff
@@ -12,7 +12,7 @@ export default {
             MATCH (resource {id: $params.resourceId})<-[:BELONGS_TO]-(report:Report {closed: false})
             WHERE resource:User OR resource:Post OR resource:Comment
             MERGE (report)<-[review:REVIEWED]-(moderator)
-            ON CREATE SET review.createdAt = $dateTime, review.updatedAt = review.createdAt
+            ON CREATE SET review.createdAt = $dateTime, review.updatedAt = $dateTime
```

Comment:
We did this to be really sure to have the same date on both on creation.

Reply:
This doesn't work with 4.4 and will make the query fail. Since its a variable it is the same value.